### PR TITLE
Dividingthecompanyreport

### PR DESCRIPTION
**Pull request by Ishrat420**
# Description

Edited this file to produce a structural company report which can be used in generating a pdf.

### CODE REVIEW

**Review by studioant — COMMENTED** [review]
Hi @Ishrat420, Only content up until `Product Overview` has been preserved in this refactoring; all product related material has been removed from the document. Is this intentional?

File: docs/company/reports/t2-2022/company-structure-obj-report.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+Each employee has chosen their own role title based on their interests, skills, experience and what
+they want to develop.
+
+Click
+[here](https://deakin365.sharepoint.com/:i:/r/sites/ThothTech2/Shared%20Documents/General/company_structure.png?csf=1&web=1&e=MJBk6i)
+to view image externally
+
+![company-structure](images/company_structure.png)
+
+---
```
Comment (studioant) [inline docs/company/reports/t2-2022/company-structure-obj-report.md:347]: Product Overview content has been removed?

**Comment by Ishrat420** [comment]
> Hi @Ishrat420, Only content up until `Product Overview` has been preserved in this refactoring; all product related material has been removed from the document. Is this intentional?

Yes it is, this .md file and each separate product .md file will be combined together and create a single Pdf file using GitHub Actions. Currently Facing some linting issue, having  a look and will fix soon.

**Review by studioant — APPROVED** [review]
This is a great initiative @Ishrat420 , well done! 🥇

**Pull request merged**